app/selectors/views.js

```diff
@@ -26,6 +26,14 @@ export const getCurrentChannelDraft = createSelector(
     }
 );
 
+export const getDraftForChannel = createSelector(
```

Comment:
This should just be a regular function. Since it has arguments, it won't be memoized anyway, and I don't think memoization is worth it since it's not doing anything complicated